cluster/saltbase/salt/kubelet/default

```diff
@@ -151,12 +151,11 @@
   {% endif -%}
 {% endif -%}
 
-{% set configure_hairpin_mode = "--configure-hairpin-mode=true" -%}
-{% if pillar.get('hairpin_mode', '').lower() == 'false' %}
-  {% set configure_hairpin_mode = "--configure-hairpin-mode=false" %}
+{% set hairpin_mode = "" -%}
```

Comment:
Yeah we figure out userspace or not within the proxier (we will fallback if iptables is wrong, and node annotations are another way to pipe kube-proxy config). Till now we've survived by isolating kubelet and kube-proxy, so I'm going to leave it that way for now.